Description: No sphinx.ext.intersphinx
 Avoids network access when building sphinx doc.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-28

--- python-keystoneclient-0.7.0.orig/doc/source/conf.py
+++ python-keystoneclient-0.7.0/doc/source/conf.py
@@ -42,7 +42,6 @@ sys.path.insert(0, os.path.abspath(os.pa
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.todo',
               'sphinx.ext.coverage',
-              'sphinx.ext.intersphinx',
               # NOTE(blk-u): Uncomment the [pbr] section in setup.cfg and
               # remove this Sphinx extension when
               # https://launchpad.net/bugs/1260495 is fixed.
